css/icons.scss

```diff
@@ -113,6 +113,11 @@
 			filter: initial;
 		}
 	}
+
+	// Patch material design icons' span wrong height
+	.material-design-icon {
+		display: contents;
```

Comment:
risky change, not sure if this is intended (in call sidebar):

before:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/277525/105506945-e6c8ca80-5cca-11eb-9549-5555b705fcaa.png">

after:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/277525/105506890-d6b0eb00-5cca-11eb-8fb9-9027f7185211.png">

Reply:
oh, apparently it also reduces the emoji icon's side in the regular messages view, so it's not about narrow panels

Reply:
This I fixed